tests/E2E/docker-files/npm-warmup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+cd /var/www/html/tests/E2E
+
+npm install  --unsafe-perm \
```

Comment:
Double space, and maybe add `npm install -g npm`?

Reply:
Why do you need `npm install -g npm` for?

Reply:
make sure npm is the latest version =)

Reply:
This file is called in the Dockerfile, and we have that command a few lines before:
```
  && npm i npm@latest -g \
```

This would be a duplicate. :)

Reply:
Huhu doesn't saw it :+1: